src/reports.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="3.0.0" type="plugin" group="api" method="upgrade">
+    <name>PLG_API_REPORTS</name>
+    <version>1.0</version>
+    <creationDate>28 May 2018</creationDate>
+    <author>Techjoomla</author>
+    <authorEmail>extensions@techjoomla.com</authorEmail>
+    <authorUrl>www.techjoomla.com</authorUrl>
+    <copyright>Techjoomla. All rights reserved.</copyright>
+    <license>http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL</license>
+    <description>Tj-Reports API plugin</description>
```

Comment:
description should also a be language constant

Reply:
done